src/shared/models/error-message.model.ts

```diff
@@ -1,8 +1,12 @@
-/**
- * Cloned from rpx-xui-webapp src/app/models/error-message.model.ts
- */
 export interface ErrorMessage {
   title: string;
   description: string;
   fieldId?: string;
+  multiple?: boolean;
```

Comment:
@connorpgpmcelroy - are these changes required? couldn't see it used anywhere in this PR.